src/datacollector/datacollector.csproj

```diff
@@ -13,7 +13,7 @@
     <AssemblyName Condition=" '$(RuntimeIdentifier)' == 'win10-arm64' ">datacollector.arm64</AssemblyName>
     <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' == 'WINDOWS_NT' ">$(TargetFrameworks);net472</TargetFrameworks>
```

Comment:
Why do we have a different min TFM depending on windows or not? I'd like to document that in the project.

Reply:
The windows code of datacollector is using something that is net472 specific, I don't remember what it is though, but you should be able to see if you downgrade. You can remove the condition for the different tfm on Linux, that probably happened because getting the whole thing build on linux was difficult in the past and I did not want to disrupt the linux build by changing it.